bin/commander.js

```diff
@@ -11,11 +11,16 @@ const chalk = require( 'chalk' );
 const fs = require( 'fs' );
 const rimraf = require( 'rimraf' );
 const SimpleGit = require( 'simple-git/promise' );
-const childProcess = require( 'child_process' );
 const Octokit = require( '@octokit/rest' );
 const os = require( 'os' );
 const uuid = require( 'uuid/v4' );
 
+/**
+ * Internal dependencies
+ */
+const { readJSONFile, runShellScript, askForConfirmationToContinue } = require( './utils' );
+const { updateCorePackages } = require( './update-core-packages' );
```

Comment:
Why this command is separate but not the others? Should we be consistent? 

Should we create a dedicated folder for this tool, now that it's split into multiple files?

Reply:
> Why this command is separate but not the others? Should we be consistent?

The main file got too big, I thought it was not a good idea to add the implementation of another command there. I think we should reorganize and split the commands into multiple files, I avoided doing this here to make reviewing easier. As soon as we land this PR I plan on proposing a PR that refactors our commands.